pkg/env/prod.go

```diff
@@ -147,6 +151,11 @@ func (p *prod) ACRResourceID() string {
 	return os.Getenv("ACR_RESOURCE_ID")
 }
 
+func (p *prod) ACRName() string {
+	resource, _ := azure.ParseResourceID(p.ACRResourceID())
```

Comment:
We already parse it in `newProd`. Can we re-use the result from `newProd`?

Reply:
good idea.  Made the change